index.html

```diff
@@ -59,9 +59,25 @@
     </header>
 
     <main>
-      <div class="overlay-image">
-        <img src="images/image2.avif" />
+      <div class="overlay-container">
+        <div class="background-image">
+          <img src="images/img2.jpg" />
+        </div>
+        <div class="overlay-text">
+          <p>Egypt</p>
+          <a href=""
+            ><button class="lets-go-button">
```

Comment:
It would be beneficial later to determine if we want to use link or button but not both.
This could apply to the navbar header item too but we can address them later. For this can you stylize so that the anchor tag looks like a button and remove <button> 👍

Reply:
Yes, I removed the button in html and add text-decoration:none for the <a> so that it removes the default underline from a link.